nemo_text_processing/inverse_text_normalization/es/taggers/ordinal.py

```diff
@@ -65,14 +79,20 @@ def __init__(self, cardinal: GraphFst):
         graph_a_suffix = (optional_numbers_in_front + ordinal_graph_a) @ cardinal_graph
         graph_er_suffix = (optional_numbers_in_front + ordinal_graph_er) @ cardinal_graph
 
-        self.graph_masc_num_no_exception = graph_o_suffix
+        self.graph_masc_num_no_exception = graph_o_suffix.optimize()
 
         # don't convert ordinals from one to nine inclusive
         graph_exception = pynini.project(pynini.union(graph_digit), 'input')
         graph_o_suffix = (pynini.project(graph_o_suffix, "input") - graph_exception.arcsort()) @ graph_o_suffix
         graph_a_suffix = (pynini.project(graph_a_suffix, "input") - graph_exception.arcsort()) @ graph_a_suffix
         graph_er_suffix = (pynini.project(graph_er_suffix, "input") - graph_exception.arcsort()) @ graph_er_suffix
 
+        if input_case == INPUT_CASED:
+            graph_exception = capitalized_input_graph(graph_exception)
```

Comment:
Where is this graph_exception used again?

Reply:
removing

Reply:
Not seeing the commit